ser/data.py

```diff
@@ -0,0 +1,21 @@
+from torch.utils.data import DataLoader
+from torchvision import datasets
+
+def get_dataloaders(batch_size, DATA_DIR, ts):
```

Comment:
another nitpick... you could argue that the DATA_DIR is only a data concern... so maybe it should just be defined in here.

Reply:
I agree, but what if not all of our data is on the same place? We would need to go to the data.py every time to change the data dir?